OpenRA.Game/Graphics/CursorSequence.cs

```diff
@@ -31,6 +31,8 @@ public CursorSequence(FrameCache cache, string name, string cursorSrc, string pa
 			Palette = palette;
 			Name = name;
 
+			Frames = cache[cursorSrc].Skip(Start).ToArray();
+
 			if ((d.ContainsKey("Length") && d["Length"].Value == "*") || (d.ContainsKey("End") && d["End"].Value == "*"))
 				Length = Frames.Length - Start;
```

Comment:
Is this even correct if we already skipped `Start` amount of frames?

Reply:
You are right. This is wrong. Fixed.

Reply:
It looks like the last push unfixed this!

Reply:
It looks like the End case will also have the same problem?

Reply:
Sorry, that happens all the time as I work from different locations/computers.